src/index.js

```diff
@@ -20,26 +20,32 @@ const TRANSFORMATIONS = {
   ],
 };
 
-module.exports = function (src, { safety = safetyLevels.SAFE, additionalTransforms = [] } = {}) {
+module.exports = function (src, options) {
   let tree = parser.parseScript(src);
+  return codegen(unminifyTree(tree, options));
+};
 
+function unminifyTree(tree, { safety = safetyLevels.SAFE, additionalTransforms = [] } = {}) {
   let transformations = [];
   for (let i = 0; i <= safety; ++i) {
     transformations.push(...TRANSFORMATIONS[i]);
   }
   transformations.push(...additionalTransforms);
 
+  let lastTree = tree;
```

Comment:
ICYMI this part was only changed because eslint was outputting a warning about assigning to a parameter.